.github/workflows/ci.yml

```diff
@@ -21,11 +21,12 @@ jobs:
       matrix:
         target:
           - riscv32imc-esp-espidf
+          - riscv32imac-esp-espidf
           - xtensa-esp32-espidf
           - xtensa-esp32s2-espidf
           - xtensa-esp32s3-espidf
         idf-version:
-          - v4.4.6
+          - v4.4.7
```

Comment:
v4.4.8 was release just a week ago as the final v4 version

Reply:
But we are still offering as default 4.4.7 in esp-idf-sys?